annotations/README.md

```diff
@@ -0,0 +1,75 @@
+# Annotations
```

Comment:
Maybe also a short update on the main README.md too? or you can link from there to here same with histograms.